app.js

```diff
@@ -195,6 +204,12 @@ app.use((err, req, res, next) => {
 	} else {
 		res.locals.message = err.message;
 	}
+
+	if (res.locals && res.locals.message.includes('ESOCKETTIMEDOUT') && err.options) {
+		const message = `ESOCKETTIMEDOUT by route: ${err.options.baseUrl + err.options.uri}`;
```

Comment:
here we don't know if err.options exist

Reply:
Nur wenn err.options existiert darf er in die Bedingung gehen. Das sollte bei dem Fehler der Fall sein. Wenn nicht soll er nichts machen.